README.md

```diff
@@ -339,6 +351,10 @@ export default MessageList = () => {
 
 *(optional)* Callback function. Called when authentication is required. If this is not supplied, `okta-react` redirects to Okta. This callback will receive [oktaAuth][Okta Auth SDK] instance as the first function parameter. This is triggered when a [SecureRoute](#secureroute) is accessed without authentication. A common use case for this callback is to redirect users to a custom login route when authentication is required for a [SecureRoute](#secureroute).
 
+#### navigate
+
+*(optional)* Router-specific function to navigate to passed relative URL programmatically.
```

Comment:
I like the addition of this prop, but I think we need to define it a bit more so users not familiar with the history will understand.  What happens if they leave out the optional prop?  What value should they pass (you have it in code samples above, but not here, and people rarely read docs start-to-finish)

Reply:
Changed to required prop